indy_node/test/validator_info/test_validator_info.py

```diff
@@ -102,28 +92,15 @@ def test_validator_info_file_get_claim_def(read_txn_and_get_latest_info,
     assert latest_info['Node_info']['Metrics']['average-per-second']['read-transactions'] > 0
 
 
-@pytest.fixture()
-def client_and_wallet(steward, stewardWallet):
-    return steward, stewardWallet
-
-
-def submitRequests(client, wallet, op):
-    req = wallet.signOp(op)
-    # TODO: This looks boilerplate
-    wallet.pendRequest(req)
-    reqs = wallet.preparePending()
-    return client.submitReqs(*reqs)[0]
-
-
-def makeGetNymRequest(client, wallet, nym):
+def makeGetNymRequest(looper, sdk_pool_handle, sdk_wallet, nym):
     op = {
         TARGET_NYM: nym,
         TXN_TYPE: GET_NYM,
     }
-    return submitRequests(client, wallet, op)
+    return sdk_submit_operation_and_get_replies(looper, sdk_pool_handle, sdk_wallet, op)
```

Comment:
here and later
are there any special conditions we do not use dedicated SDKs build_req* functions?

Reply:
No